pyoctoprintapi/octoprint_client.py

```diff
@@ -98,4 +100,34 @@ async def pause_job(self) -> None:
 
     async def resume_job(self) -> None:
         _LOGGER.debug("Sending resume job command")
-        await self._api.issue_job_command(JOB_COMMAND_PAUSE, JOB_COMMAND_PAUSE_RESUME)
\ No newline at end of file
+        await self._api.issue_job_command(JOB_COMMAND_PAUSE, JOB_COMMAND_PAUSE_RESUME)
+
+    async def disconnect(self) -> None:
+        _LOGGER.debug("Disconnecting from printer")
+        await self._api.issue_connection_command("disconnect")
+
+    async def connect(self, port: Optional[str] = None, baud_rate: int = None, printer_profile: str = None, save: bool = None, auto_connect: bool = None) -> None:
```

Comment:
Should all this parameters be marked optional seeing as None is the default
```suggestion
    async def connect(self, port: Optional[str] = None, baud_rate: Optional[int] = None, printer_profile: Optional[str] = None, save: Optional[bool] = None, auto_connect: Optional[bool] = None) -> None:
```

Reply:
Hah! I did this originally, but thought it was wrong. I stopped using python before type hinting became a thing.